core/constraint-streams/src/main/java/ai/timefold/solver/constraint/streams/bavet/bi/Group0Mapping1CollectorBiNode.java

```diff
@@ -13,7 +13,8 @@ final class Group0Mapping1CollectorBiNode<OldA, OldB, A, ResultContainer_>
     public Group0Mapping1CollectorBiNode(int groupStoreIndex, int undoStoreIndex,
             BiConstraintCollector<OldA, OldB, ResultContainer_, A> collector,
             TupleLifecycle<UniTuple<A>> nextNodesTupleLifecycle, int outputStoreSize, EnvironmentMode environmentMode) {
-        super(groupStoreIndex, undoStoreIndex, null, collector, nextNodesTupleLifecycle, environmentMode);
+        super(groupStoreIndex, undoStoreIndex,
```

Comment:
Note for reviewers: there is *nothing interesting* in these `Group*` changes. Only mechanical updates, pure boilerplate.

All the interesting stuff is typically in `Abstract*` classes.